components/gitpod-db/src/typeorm/entity/db-workspace-cluster.ts

```diff
@@ -97,4 +98,29 @@ export class DBWorkspaceCluster implements WorkspaceCluster {
     // This column triggers the periodic deleter deletion mechanism. It's not intended for public consumption.
     @Column()
     deleted: boolean;
+
+    @Column({
+        type: "simple-json",
+        transformer: (() => {
```

Comment:
I think we can use `Transformer.SIMPLE_JSON`` here.
But it would be even better to use the `JSON` type on DB level.
See for example [`DBWorkspaceInstance#gitstatus`](https://github.com/gitpod-io/gitpod/blob/2a9a442c7e3bc39b04bd2016f1a245144459e0c1/components/gitpod-db/src/typeorm/entity/db-workspace-instance.ts#L80-L84)